### PR TITLE
show the owner in repo_tree_view

### PR DESCRIPTION
**Pull request by Chilledheart**
Screen shot for private shares:

screen shot 2015-01-16 at 12 48 53

Screen shot for group shares:

screen shot 2015-01-16 at 12 50 43

### CODE REVIEW

File: src/ui/repo-item-delegate.cpp [hunk 1]
```diff
@@ -218,6 +222,27 @@ void RepoItemDelegate::paintRepoItem(QPainter *painter,
                       &repo_desc_rect);
     painter->restore();
 
+    // Paint repo sharing owner for private share
+    // TODO: show only for private share
+    if (item->repo().owner != seafApplet->accountManager()->accounts().front().username) {
```
Comment (lins05) [inline src/ui/repo-item-delegate.cpp]: `accountManager()->currentAccount()` can help here.

**Comment by lins05** [comment]
We should only display the name part the email address,  for example only show "foo" for "foo@bar.com"

File: src/ui/repo-item-delegate.cpp (hunk 1 above)
Comment (Chilledheart) [inline src/ui/repo-item-delegate.cpp]: I know what, but `currentAccount` returns a new (and unnecessary) copy of Account while using `accounts().front()` doesn't (only reference)

**Comment by Chilledheart** [comment]
Well, I am pretty con to that. It looks like no difference between group owner and person owner in that way.

**Comment by Chilledheart** [comment]
Updated, no more group owner shown. cc @freeplant 

screen shot 2015-01-19 at 15 15 56

**Comment by Chilledheart** [comment]
The owner should be shown only in the category "private share" now.

File: src/ui/repo-item-delegate.cpp [hunk 2]
```diff
@@ -218,6 +222,27 @@ void RepoItemDelegate::paintRepoItem(QPainter *painter,
                       &repo_desc_rect);
     painter->restore();
 
+    // Paint repo sharing owner for private share
+    if (static_cast<RepoCategoryItem*>(item->parent())->categoryIndex() ==
```
Comment (lins05) [inline src/ui/repo-item-delegate.cpp]: We can use `item->repo().isShareRepo()` here.

Comment (Chilledheart) [inline src/ui/repo-item-delegate.cpp]: Yeah, but we want to show the owner precisely in the private shared category, not outside.

Comment (lins05) [inline src/ui/repo-item-delegate.cpp]: That's fair enough.

**Pull request merged**